project-management/src/main/java/life/qbic/projectmanagement/application/VirtualThreadScheduler.java

```diff
@@ -1,4 +1,4 @@
-package life.qbic.datamanager;
+package life.qbic.projectmanagement.application;
```

Comment:
I think the VirtualThreadScheduler can be in application commons even. 
It is not bound to any domain nor to the ui and I think putting it in the globally accessible package makes the most sense. What do you think?

Reply:
I opt for delaying it, until we see that other modules also need this concept :)